SR2025/2024-09-26-kickstart-details.md

```diff
@@ -0,0 +1,56 @@
+---
+to: All currently signed-up teams
+subject: SR2025 Kickstart details
+---
+
+Hi!
+
+Kickstart is on October 12th, only a couple weeks away! So it's time to give you a bit more info about the event.
+
+Kickstart is the big kick-off event for the competition. We'll firstly be announcing the game for which you'll be building your robots, alongside some additional challenges to complete along the way. Finally, we'll present the 'Microgames', a series of tasks to get acquainted with your new kits. Our blueshirts (our lovely volunteers) will be available to mentor your teams throughout.
```

Comment:
Feels a bit odd to have "firstly" then "finally" for only two items? Do we want to also promise to e.g: have the tech day dates?